irr/src/COpenGLCoreTexture.h

```diff
@@ -170,6 +170,19 @@ class COpenGLCoreTexture : public ITexture
 			return;
 		}
 
+#ifndef IRR_COMPILE_GL_COMMON
+		// TODO explain this correctly
+		if (InternalFormat == GL_BGRA && Driver->Version.Major >= 3) {
+			InternalFormat = GL_RGBA8;
+			PixelFormat = GL_RGBA;
+		}
```

Comment:
`post_processing_texture_bits = 8` works now on my device with OpenGL ES 3.2

Reply:
what's the performance difference between 8/10/16?